File: Data Structures & Algorithms/duplicate-integer/submission-0.go
func hasDuplicate(nums []int) bool {
	mynumsMap := make(map[int]int)
	for _, e := range nums {
		if _, ok := mynumsMap[e]; ok {
			return true
		} else {
			mynumsMap[e] = 1
		}
	}
	return false
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        out = []
        temp = []
        for i in range(len(nums)):
            res = 1
            for j in range(i+1,len(nums)):
                res *= nums[j]
            for k in temp:
                res *= k
            out.append(res)
            temp.append(nums[i])
        
        return out